---
title: "Work Package 4: Phylogenetics"
keywords: about
last_updated: October 5, 2017
tags: [wp4]
summary:
sidebar: artic_sidebar
permalink: wp4-phylogenetics.html
toc: false
folder: artic
---


# Aims

{% include links.html %}

1.Generating a real-time temporal and evolutionary overview of the epidemic

We aim to develop an efficient ‘real-time’ approach that can automatically update the time-measured phylogeny as sequence data is generated. The concept of this is depicted in the figure below. In addition to the approach implemented in [nextstrain](www.nextstrain.org/), we will explore a Bayesian inference procedure adopting the [BEAST](http://beast.community) framework that can rapidly converge to then posterior distribution of phylogenies for an expanded sequence data set.

{% include image.html file="realTime+Trees-Maps.png" caption="" %}

2.Revealing geographical dispersal and mapping genetic and phenotypic changes

We aim to use phylogeographic models to reconstruct the geographical dissemination of viruses as the epidemic progresses. For this purpose, we can adopt discrete and continuous diffusion models as well 'population structure' models based on coalescent or birth-death process. Thes analyses will be used to provide detailed, practical and interpretable visualizations in WP5 (as illustrated in the figure) as well as detailed information about the overall process (i.e., numbers of movements, distances travelled, locations that are hubs of viral migration). 

Increasingly, viral genomes will be associated with viral phenotypic (e.g. antigenic measurements) or clinical data on the individual cases (e.g., viral load or mortality, or infant head size). Such data can be incorporated similarly to geographical data. Over the course of an outbreak such approaches could provide warnings about changes in clinical manifestation that may be due to changes in the virus.

3.Approaches for incorporating within-host variation into phylodynamic analysis

As part of this aim, we will investigate ways of modeling changes in nucleotide state frequencies over evolutionary time as a way of informing phylogenetic estimates of transmission chains.

4.Testing hypotheses of epidemiological linkages between cases

5.Evaluation of system

{% include presentations.html %}


