---
title: "ARTIC-EBOV-phylogeneticsSOP-v1.0.0 | Ebola virus real-time phylogenetics"
keywords: protocol
layout: document
last_updated: May 18, 2018
tags: [protocol]
summary:
permalink: ebov/ebov-phylogenetics-sop.html
folder: ebov
title_text: "Ebola virus bioinformatics protocol"
subtitle_text: "Nanopore | bioinformatics"
document_name: "ARTIC-EBOV-phylogeneticsSOP-v1.0.0"
version: v1.0.0
creation_date: 2018-05-26
forked_from: 
author: Andrew Rambaut, Philippe Lemey
citation: "Loman *et al.* In Prep."
---

{% include callout.html
type='default'
content='**Overview:** '
%}

<br />

This document is part of the Ebola virus Nanopore sequencing protocol package:
: [http://artic.network/ebov/](http://artic.network/ebov/)

<br /><br /><br />

{% include wellcome-trust.html %}

<div class="pagebreak"> </div>

## Preparation

Set up the computing environment as described in this document: [ebov-it-setup-1.0.0](ebov-it-setup.html)

This protocol also assumes that the setup and installation of the bioinformatics protocol has been performed as described in this document: [ebov-bioinformatics-sop-1.0.0](ebov-bioinformatics-sop.html) .

### Installing software

Activate the ARTIC Conda environment:

```
source activate artic
```

Install the phylogenetics packages required:

```
conda install muscle phyml ete3
```

### Reference genomes

An alignment of 35 complete or nearly-complete genomes spaning 1976-2014 is available. This has representitives from many of the Middle Africa outbreaks in DRC, Gabon and Republic of Congo and 3 from the West African outbreak in Guinea in 2014. This set is intended to provide a framework in order to place new, previously uncharacterised, outbreak sequences.

[35 EBOV genome alignment - FASTA file.](https://github.com/artic-network/ebov/blob/master/reference_genomes/ebov-reference-genomes-35.fasta)

This is a FASTA format file which contains a multiple alignment of the 35 genomes. 

## Building a multiple alignment

<!--
Merge the EBOV genome reference sequence file with the new consensus sequence:

```
cat new_consensus.fasta ebov-reference-genomes-35.fasta > unaligned.fasta
```

Use [MUSCLE](http://www.drive5.com/muscle/) multiple alignment software to align the genomes:

```
muscle -in unaligned.fasta -fastaout aligned.afa -htmlout aligned.html
```

> **Note:** With many sequences this step may become quite slow. There are faster, more approximate ways of doing this - i.e., using MUSCLE's feature for adding a sequence to an existing alignment.
-->

Use [MUSCLE](http://www.drive5.com/muscle/) multiple alignment software to align the new genome consensus sequences to the existing reference genome alignment:

```
muscle -profile -in1 ebov-reference-genomes-35.fasta -in2 new_genomes.fasta -fastaout new_alignment.afa
```

This methods keeps the existing alignment and pair-wise aligns the new sequence to it. This is much quicker than doing a full multiple alignment but could be problematic if the new genome is divergent from all the reference genomes.  

A simple Neighbor Joining tree can be generated by MUSCLE:

```
muscle -maketree -cluster neighborjoining -in new_alignment.afa -out ebov.tree 
```

This method uses a simple model of evolution but is a useful way of getting a quick evaluation of the new sequence and the alignment.

[ETE3](http://etetoolkit.org/documentation/tools/) can be used to open a window to view the resulting tree:

```
ete3 view -t ebov.tree
```



<div class="pagebreak"> </div>
## Phylogenetic analysis

